.github/workflows/generateChangelog.yml

```diff
@@ -14,18 +14,15 @@ jobs:
           ref: 'develop'
           ssh-strict: false
           token: ${{ secrets.IDEE_GH_TOKEN }}
-          fetch-depth: '0'
+          fetch-depth: '0' # Fetch all tags to prepare for tag retrieval
```

Comment:
nit This should probably say the call to fetch all tags fails if we don't limit the depth 🤷